src/main/java/com/programmers/java/engine/calculator/Calculator.java

```diff
@@ -0,0 +1,7 @@
+package com.programmers.java.engine.calculator;
+
+import com.programmers.java.engine.model.Expression;
+
+public interface Calculator<T> {
+    T calculate(Expression expression);
```

Comment:
제네릭 사용할때 이렇게 변경하면 T에 어떤 것이 들어올지 알기 좀 더 좋지 않을까 싶습니다!
```suggestion
public interface Calculator<T extends Number> {
    T calculate(Expression expression);
```